valve-firmware/valve-firmware.spec

```diff
@@ -1,13 +1,14 @@
 Name: valve-firmware
-# There are two source packages. Set the RPM version to mirror whichever source is newer.
+# There are two source packages that use a date version. Set the RPM version to mirror whichever source is newer.
 Version: 20231113.1
-Release: 1%{?dist}
+Release: 3%{?dist}
 Summary: Linux firmware files for the Steam Deck OLED
 License: GPL+ and GPLv2+ and MIT and Redistributable, no modification permitted
 URL: https://steamdeck-packages.steamos.cloud/archlinux-mirror/jupiter-main/os/x86_64/
 Source0: https://steamdeck-packages.steamos.cloud/archlinux-mirror/jupiter-main/os/x86_64/linux-firmware-neptune-jupiter.%{version}-1-any.pkg.tar.zst
 Source1: https://steamdeck-packages.steamos.cloud/archlinux-mirror/jupiter-main/os/x86_64/linux-firmware-neptune-20230121.1f01c88-1-any.pkg.tar.zst
-BuildArch: noarch
+Source2: https://steamdeck-packages.steamos.cloud/archlinux-mirror/jupiter-main/os/x86_64/steamdeck-dsp-0.49-1-any.pkg.tar.zst
+BuildArch: x86_64
```

Comment:
RPM automatically detects that some of the installed binaries for audio support are 64-bit and fails so we can no longer user `noarch` here. This should be fine as all Steam Decks are x86_64 anyways. There's no reason to install this package on Arm or other architectures.